CondTools/CTPPS/test/write-ctpps-pixel-gainscalibrations_cfg.py

```diff
@@ -0,0 +1,75 @@
+import FWCore.ParameterSet.Config as cms
+import sys,os
+
+arguments = sys.argv
+infile = "Gains.root"
+tagname = "CTPPSPixelGainCalibrations_test"
+sqlitename = "ctppsgains_test.db"
+runnumber = 1
+doDummy = False
+gainlo = 0.0
+gainhi = 100.0
+minNp = 3
+if len(arguments)<3:
+    print "using default values"
```

Comment:
@malbouis in this one

Reply:
@fabiocos , I updated the files. Please let me know if there is anything else I might have missed. Thanks!